driver/bpf/fillers.h

```diff
@@ -3157,7 +3157,9 @@ FILLER(sys_io_uring_setup_x, true)
 	flags = io_uring_setup_flags_to_scap(params.flags);
 	sq_thread_cpu = params.sq_thread_cpu;
 	sq_thread_idle = params.sq_thread_idle;
+#ifdef IORING_FEAT_SINGLE_MMAP	
 	features = io_uring_setup_feats_to_scap(params.features);
+#endif	
```

Comment:
Wouldn't this potentially leave `features` uninitialized when `IORING_FEAT_SINGLE_MMAP` is not defined? The verifier will probably not be happy about this.

Reply:
thank you as always @Molter73 :heart:

Reply:
Great catch! :rocket: :bug:

Reply:
I know this doesn't matter too much in the grand scheme of things, but how about we move the ifdef to inside `io_uring_setup_feats_to_scap`? I'd say it's clearer why that ifdef is there

Reply:
hi @gnosek, i think we cannot move this `#ifdef` inside `io_uring_setup_feats_to_scap` because we will have a compilation error with `params.features`, the struct `io_uring_params` doesn't contain the `features` field if the flag is not defined